src/store/components/AdminItemPage/index.tsx

```diff
@@ -129,6 +130,12 @@ const AdminItemPage: React.FC<AdminItemPageProps> = (props) => {
     discountPercentage: (!item?.hasVariantsEnabled && item?.options[0].discountPercentage.toString()) || '',
   };
 
+  // converts uuid of a collection to { label: ..., value: ... }
+  const getOptionFormat: (string) => Option = (uuid: string) => {
```

Comment:
Fixed with a better long-term solution in #608, if we revert this code back, we can merge as a solution to the other bug being resolved.